rivers/books_drupal.json

```diff
@@ -0,0 +1,131 @@
+{
+  "type" : "remote",
+  "remote" : {
+    "urlGetDocuments"               : "http://developer-drupal.web.stage.ext.phx2.redhat.com/drupal/books",
+    "listDocumentsMode"             : "simple",
+    "spacesIndexed"                 : "MAIN",
+    "timeout"                       : "20s",
+    "indexUpdatePeriod"             : "10m",
+    "maxIndexingThreads"            : 1
+  },
+  "index" : {
+    "index"                         : "data_jbossdeveloper_book",
+    "type"                          : "jbossdeveloper_book",
+    "remote_field_document_id"      : "nid",
+    "field_document_id"             : "sys_id",
+    "fields": {
+      "authors": {"remote_field": "prep_sys_authors"},
```

Comment:
Not sure if intentionally you prep sys authors but then put them in authors field instead of sys_authors.